crates/storage/db/src/tables/mod.rs

```diff
@@ -415,6 +416,11 @@ table!(
     ( SyncStageProgress ) StageId | Vec<u8>
 );
 
+table!(
+    /// Stores the highest pruned block number.
+    ( PruneStage ) StageId | BlockNumber
+);
```

Comment:
i dream of collapsing `SyncStage`, `SyncStageProgress` & `PruneStage` into a single table